run.py

```diff
@@ -0,0 +1,6 @@
+# entrypoint
+from app import app
+
+
+if __name__ == "__main__":
```

Comment:
Add a port number on which the server should run. Ideally it should be an environment variable like PORT or 3000 if the env is not set.

Also debug should be false if app is running on production

Reply:
I will work on this